guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -26,6 +26,26 @@ include::topics/upgrading_satellite_server_parent.adoc[leveloffset=+2]
 // Upgrading a Connected Satellite Server
 include::topics/upgrading_a_connected_satellite_server.adoc[leveloffset=+3]
 
+ifdef::satellite[]
+// Migrating to a New Server
+include::common/modules/proc_migrating-to-a-new-server.adoc[leveloffset=+2]
+endif::[]
```

Comment:
This now renders at a wrong level:
![image](https://user-images.githubusercontent.com/94284/174980075-507c85e4-de30-4b5c-b05b-1fa734d29df9.png)

I would have expected to see one topic document included here, which in turn includes the modules. But I am not 100% familiar with docs guidelines so will defer to others for actual best practices.

Reply:
@evgeni Yes, that's how I originally had it laid out, but when I was reviewing with a teammate, I realized I couldn't put a list of procedures within a procedure.

Reply:
but now the chapter structure is completely wrong?